### PR TITLE
Update the Products section in the footer

### LINKED ISSUE #10998: update 'Products' in footer
In the Products section of the site footer, please...

- Remove Lockwise
- Add [VPN](https://www.mozilla.org/products/vpn/#pricing)
- Add [Relay](https://relay.firefox.com)

### PR DESCRIPTION
**Pull request by bobsilverberg**
Fixes #10998

### CODE REVIEW

File: src/amo/components/Footer/index.js [hunk 1]
```diff
@@ -204,39 +204,50 @@ export class FooterBase extends React.Component<InternalProps> {
...
+                  className="Footer-browsers-link"
+                  href={`https://www.mozilla.org/firefox/browsers/${makeQueryStringWithUTM(
                     {
                       utm_content: 'footer-link',
                       utm_campaign: null,
                     },
                   )}`}
                 >
-                  Lockwise
+                  Browsers
```
Comment (diox) [inline src/amo/components/Footer/index.js:215]: We were already not translating it before, but... should `Browsers` be left untranslated ?

Comment (bobsilverberg) [reply src/amo/components/Footer/index.js:215]: I have asked about this on Slack.

Comment (bobsilverberg) [reply src/amo/components/Footer/index.js:215]: It seems that the answer is "yes, leave it untranslated".

**Review by diox — APPROVED** [review]

File: src/amo/components/Footer/index.js (hunk 1 above)
Comment (diox) [reply src/amo/components/Footer/index.js:215]: :shrug:

**Pull request merged**